src/pages/FolderPage.js

```diff
@@ -1,13 +1,45 @@
-import FolderHeader from "../component/FolderHeader";
-import InfoLoader from "../component/InfoLoader";
-import FolderMain from "../component/FolderMain";
-function FolderPage() {
-  const { folderInfo, userFolderList } = InfoLoader();
 
+import FolderHeader from "../component/folder/FolderHeader";
+
+import FolderMain from "../component/folder/FolderMain";
+import { useState } from "react";
+import ContainerModal from "../component/utilModal/ContainerModal";
+function FolderPage({ folderInfo, userFolderList }) {
+  const [modalOpen, setModalOpen] = useState(false);
+  const [selectedModalOption, setSelectedModalOption] = useState("");
+  const [folderId, setFolderId] = useState();
+  const [inputLinkValue, setInputLinkValue] = useState("");
+
+  const handleModalOpen = (boolean, modalOption, inputLinkValue = "") => {
```

Comment:
데이터형을 변수이름으로 사용하는 것은, 의미를 알수없게하고 (타입에 대한 정보전달은 타입스크립트에서 해결됩니다) 오래된 안티패턴이라서 제안드립니다.
```suggestion
  const handleModalOpen = (isOpen, modalOption, inputLinkValue = "") => {
```